### DISCUSSION TITLE
2024.09.06 Town Hall

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/virtualcoffee.io#1225] [Town Halls]
Hello Virtual Coffee members!

Below are the minutes from the VC Town Hall held on Friday, September 6th. Please add any thoughts, comments, questions, etc. in a thread on this post and we will continue discussion here. If you wish to provide feedback anonymously, you can use the [Virtual Coffee Town Hall form](https://forms.gle/f7ngSFTqydLbJ4hA8) at any time and your submission will be addressed in the subsequent meeting.

Our next Town Hall is currently scheduled for Friday, December 6th at 1:00 PM ET.

---

# Virtual Coffee Town Hall – September 6, 2024

## Intro

We’ve been doing this since April 2020, and we’ve loved the growth we’ve seen. Virtual Coffee's mission is to be a welcoming tech community that allows room for growth and mentorship at all levels, and to create meaningful opportunities for learning, leadership, and contribution for everyone.

### Important Links

- Code of Conduct: https://virtualcoffee.io/code-of-conduct/
- COC Violation Form: https://virtualcoffee.io/report-coc-violation/
- How to get Involved: https://virtualcoffee.io/resources/virtual-coffee/get-involved

### Summary of Community Changes

1. We did the [Welcoming Community Challenge](https://virtualcoffee.io/monthlychallenges/july-2024) in July, the [Photography Challenge](https://virtualcoffee.io/monthlychallenges/aug-2024) in August, and now the [Preptember Challenge](https://virtualcoffee.io/monthlychallenges/sept-2024) this month!
2. Arvind created the [task-challenges](https://github.com/Virtual-Coffee/task-challenges) repo to help members acquire skills used in the workplace. Reach out to Arvind if you’re interested in helping with this initiative! (Give Arvind the floor for 2-3 minutes)
3. Jenna and Andy are [gathering interest for a Study Group for Practical Deep Learning for Coders](https://virtual-coffee-group.slack.com/archives/C029MJ4E327/p1725032329793609). They plan to do a check-in every other Tuesday at 5:00 PM ET as well as run an asynchronous GitHub Discussion. Respond in Slack if you’re interested!

    > Jenna and Andy will be learning deep learning and machine learning through [course.fast.ai](https://course.fast.ai/).
    > It is a _free course designed for people with some coding experience, who want to learn how to apply deep learning and machine learning to practical problems._
    > They'll meet up every other Tuesday (afternoons US ET) to chat about what they're learning. Join them in their quest to learn all the... deep learning and AI things! Stop by `#learning-together` and `#dev-ml-data-ai` to find out more.

4. We will begin polling later today for a secondary time to host a Pack Hunt/Accountabilibuddies session. This will be hosted by Lauren, who led the Cracking the Coding Interview book club. She’s based in China so hopefully the time will accommodate many of our members in that hemisphere. Let Lauren know if you’re interested in co-hosting!
5. We’ve decided to move the website from Remix to Next.js for the overhaul.
**Why?** It will be a lot of work to update the current site on Remix. Next.js is the first framework React points to when getting started so the hope is to lower the barrier to entry. Remix has gone out of date.
**The Plan.** Get a lot of it started and see where we are. Dan wants to move to TailwindCSS for styling after the migration to Next.js.

### Outstanding Open Items

1. We will be open sourcing the VC-Contributors repo in the next few months.
**Why?** It gives insight into our internal processes and documentation.
**The Goal.** Open sourced by September 30th.
2. Next month we’ll be doing Hacktoberfest as our monthly challenge.

### Cursor.ai

- [Cursor.ai](https://www.cursor.com/) is an AI code editor
- A member suggested that we use this as a tool to help switch the site from Remix to Next.js
- Another member vouched that Cursor.ai is "a major leap forward compared to the other ones I have tried"

### Speed Networking Events

- A member suggested that we hold speed networking events in which VC members are paired and have a limited amount of time to chat with each other
- How it would work:
  - We would use an open-source version of the Donut Slack app
  - We would use Slack's one-on-one Huddle feature
  - We would need a dedicated Lead to organize the events
- We have done this in the past and may still have guides from then
  - Bekah will try to find them

## Action Items

- [ ] Bekah – Look to see if we have documentation on when we last did "speed networking" in VC

## Next Meeting's Agenda

- Summarize community changes since last meeting
- List any outstanding open items
- Open the floor for community questions, feedback, and discussion
- Address any items submitted through the [Virtual Coffee Town Hall form](https://forms.gle/f7ngSFTqydLbJ4hA8) that weren’t addressed during the course of open discussion

**Comment by albendz** [comment]
The Donut matching style and speed networking is a great idea and I've enjoyed participating in other Slack groups. I think it's worth taking a poll to see who would participate to make sure there are enough active members so that we're not matched with the same people. Sorry to be redundant if you've already done this research.

The experience sharing group and directory is also a great idea!

For Remix to Next.js, is this a big enough project to be part of hacktoberfest or will the AI tool take care of it pretty quickly?
